src/viam/components/movement_sensor/client.py

```diff
@@ -3,7 +3,7 @@
 from grpclib.client import Channel
 
 from viam.components.movement_sensor.movement_sensor import MovementSensor
-from viam.proto.common import DoCommandRequest, DoCommandResponse, Geometry
+from viam.proto.common import DoCommandRequest, DoCommandResponse, Geometry, GetReadingsRequest, GetReadingsResponse
```

Comment:
I see some import errors for these values in the test; I'm guessing `GetReadingsRe*` types were added to the `api` repo but they have yet to be automatically imported to the Python SDK.

Reply:
Ya exactly -- this has to be merged in after the API changes get merged